### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by jleslie85**

### CODE REVIEW

File: stable/README.md [hunk 1]
```diff
@@ -103,12 +104,12 @@ In order to use this helm chart locally you will need to first update the depend
...
 
-You can now install the chart:
+Install the chart:
 ```
-helm install stable/insights [--generate-name | --name releaseName] [--set parameter] [--values myvalues.yaml]
+helm install [name] stable/insights [--generate-name] [--set parameter] [--values myvalues.yaml]
 ```
 
-### Grant Red Had OpenShift privileges
+### Grant Red Hat OpenShift privileges
```
Comment (sivanov-nuodb) [inline stable/README.md:112]: Good catch!

**Review by sivanov-nuodb — APPROVED** [review]
Looks good to me. Thanks!

File: stable/README.md [hunk 2]
```diff
@@ -146,13 +147,14 @@ helm install nuodb-insights/insights --generate-name -n nuodb \
...
 ## Accessing NuoDB Insights
 
-By default Grafana will be available within the Kubernetes cluster via ClusterIP service. One way to access Grafana dashboards is to use port forwarding and navigate to http://localhost:8080/.
+By default, the NuoDB Insights Grafana dashboard WebUI will be available within the Kubernetes cluster via ClusterIP service. One way to access the WebUI dashboards is to use port forwarding and navigate your web browser to http://localhost:8080/.
 
 ```
-kubectl port-forward service/<release-name>-grafana 8080:80
+kubectl port-forward service/<name>-grafana 8080:80
 ```
+For `<name>`, use the value used in your `helm install` command for the `name` argument.
```
Comment (vegichan) [inline stable/README.md:155]: you picked `insights`. We should just go with that.

**Review by vegichan — APPROVED** [review]
looks good

File: stable/README.md (hunk 2 above)
Comment (jleslie85) [reply stable/README.md]: Sounds good, goes along with the simplify theme : ) 
Done.

**Pull request merged**